packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.controller.js

```diff
@@ -46,7 +63,12 @@ export default class VolumeBackupCreateController {
       volumeOption: selectedVolumeType,
     } = this.volumeBackupModel.selected;
 
-    return selectedVolume && selectedVolumeType && this.volumeBackupModel.name;
+    return (
+      selectedVolume &&
+      selectedVolume.attachedTo.length === 0 &&
+      selectedVolumeType &&
+      this.volumeBackupModel.name
```

Comment:
**nitpick:** The expression can be simplified.
```suggestion
      selectedVolume?.attachedTo.length === 0 &&
      selectedVolumeType &&
      this.volumeBackupModel.name
```

Reply:
done